.github/workflows/CompatHelper.yml

```
@@ -0,0 +1,16 @@
+name: CompatHelper
+on:
+  schedule:
+    - cron: '00 00 * * *'
```

Or something similar? I feel like crons and regex always need some explanation for them.

```suggestion
    - cron: '00 00 * * *'  # Everyday at midnight
```